lib/tools/system-calls.js

```diff
@@ -771,42 +772,50 @@ systemCallMethods.waitForDevice = async function (appDeviceReadyTimeout = 30) {
   });
 };
 
+/**
+ * Tries to run privileged shell commands without root, otherwise elevates privileges and run them again
+ *
+ * @param {args} array - Shell arguments commands
+ */
+systemCallMethods.runPrivilegedShell = async function (args) {
+  try {
+    this.isRoot = false;
```

Comment:
I'd rather simply return the result from the function. Using instance properties for this purpose creates too many unnecessary dependencies.